main.py

```diff
@@ -190,16 +275,20 @@ def another_content_type_handler(message: telebot.types.Message):
     bot.reply_to(message, "Сообщения этого типа не поддерживаются. Свяжитесь с @kolayne, чтобы добавить поддержку")
 
 
-@bot.callback_query_handler(func=lambda call: True)
+# Invalid callback query handler
+@bot.callback_query_handler(func=lambda call: does_raise(json.loads, args=(call.data,), expected=json.JSONDecodeError,
+                                                         reraise_other=False) or
```

Comment:
I would like this to not only catch errors from `json.loads` but also verify that its return-value is a dictionary